programs/ziti-edge-tunnel/netif_driver/linux/tun.c

```diff
@@ -24,8 +29,25 @@
 #define IP_BIN "/sbin/ip "
 #endif
 
+#define CHECK_UV(op) do{ int rc = op; if (rc < 0) ZITI_LOG(ERROR, "uv_err: %d/%s", rc, uv_strerror(rc)); } while(0)
+
 #define RESOLVECTL "resolvectl"
 
+static struct {
+    char tun_name[16];
```

Comment:
This would be a good place to use `IFNAMSIZ`

Reply:
:+1: